cmd/ipfs-cluster-ctl/graph.go

```diff
@@ -6,8 +6,8 @@ import (
 	"io"
 	"sort"
 
+	dot "github.com/kishansagathiya/go-dot"
```

Comment:
```suggestion
	dot "github.com/zenground0/go-dot"
```

after https://github.com/ZenGround0/go-dot/pull/4 gets in